Documentation/contributing/testing/ci.rst

```diff
@@ -153,103 +153,73 @@ packet.net before the branch is merged.
++==================+==========================+
+| master           | test-me-please           |
++------------------+--------------------------+
+| v1.9             | test-backport-1.9        |
++------------------+--------------------------+
+| v1.8             | test-backport-1.8        |
++------------------+--------------------------+
+| v1.7             | test-backport-1.7        |
++------------------+--------------------------+
+
+On top of these shorthands for triggering required jobs, one may use any
+of these additional triggers to either trigger optional jobs, or manually
+re-trigger a single job that previously failed (in case of a flake):
+
++--------------------+---------------------------------------------------------------------------+
+| Trigger phrase     | PR job(s) triggered                                                       |
++====================+===========================================================================+
+| test-X.XX-4.9      | Specific K8s version ``X.XX`` on Kernel 4.9                               |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-4.19     | Specific K8s version ``X.XX`` on Kernel 4.19                              |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-net-next | Specific K8s version ``X.XX`` on Kernel net-next                          |
++--------------------+---------------------------------------------------------------------------+
+| test-gke           | GKE test (runs on K8s version currently marked as "stable" in GKE)        |
++--------------------+---------------------------------------------------------------------------+
+| test-runtime       | Runtime test                                                              |
++--------------------+---------------------------------------------------------------------------+
+| test-upstream      | Upstream test (note: K8s and Kernel versions vary depending on PR branch) |
++--------------------+---------------------------------------------------------------------------+
+| test-4.9           | Latest K8s version on Kernel 4.9                                          |
++--------------------+---------------------------------------------------------------------------+
+| test-4.19          | Latest K8s version on Kernel 4.19                                         |
++--------------------+---------------------------------------------------------------------------+
+| test-net-next      | Latest K8s version on Kernel net-next                                     |
++--------------------+---------------------------------------------------------------------------+
+| test-missed-k8s    | All non-required K8s versions on Kernel 4.9 **for ``master`` PRs**        |
++--------------------+---------------------------------------------------------------------------+
+
```

Comment:
Thanks for this!

A nice addition that I, for one, would appreciate, is a way to mindlessly match the phrase to the failure I'm seeing on the PR.

![image](https://user-images.githubusercontent.com/221481/108356808-e568c000-71ec-11eb-98be-98aee2a9872b.png)

For example, it would be great if I could do a Ctrl-F with the failure I'm seeing on the PR (see above image) to the documentation page to find quickly how I can re-trigger it.

Would it make sense to add another column here that depicts how each of these tests would appear on a PR?

Also, I'm not exactly sure which one is the Cilium-ginkgo-GKE... I'm guessing it's the test-gke, but I don't think it's 100% clear.

An alternative to another column would be to make the names that appear on the PR to match the trigger phases or vice-versa. For example, triggering Cilium-Ginkgo-GKE could be done by `test-Cilium-Ginkgo-GKE`

Reply:
AFAIU, all tests can now be triggered with `test-[K8s version]-[kernel version]`. So K8s-1.20-kernel-4.9 is `test-1.20-4.9`; K8s-1.13-Kernel-netnext is `test-1.13-netnext`. The one exception is the single GKE job which is triggered with... you guessed it, `test-gke`.

Reply:
Yea, all of these should be documented in the table as well. With that said, I understand the sentiment that there is currently no way to link a PR status to a trigger. I would personally be favorable to adding individual triggers to the status' names, e.g. `K8s-1.20-kernel-4.9` could become `K8s-1.20-kernel-4.9 (test-1.20-4.9)`. We could also take the opportunity to normalize statuses and job names.

The only problem with that is that we need to update GitHub required jobs accordingly, and this is a two-step process (required jobs are known by name, thus if the name changes one has first to trigger it once and make it known to GH, then untick the old one and retick the new one). So, all outstanding PRs will have their requirements changed and potentially need to re-pass all required checks.

I don't have the access rights for this, but @nebril does. If there is demand we could plan it in advance and then pick a specific time for the transition.

Reply:
Out of curiosity, is it not possible to have multiple triggers so that we can keep both the old and the new?

Reply:
Can you clarify which triggers you had in mind? Most jobs can already be triggered by multiple different phrases, and I actually don't think any of the old triggers got removed 🤔

Reply:
Makes sense. Did not not have specific triggers in mind.

Reply:
For information, Maciej and I have convened to have a look at renaming/standardizing PR checks on Monday, with the idea of adding triggers directly in the check name. This is on purely Jenkins' side and will be done outside of this PR.